packages/@sanity/components/src/dialogs/PanePopover.js

```diff
@@ -0,0 +1,75 @@
+/* eslint-disable complexity */
```

Comment:
It doesn't appear to me that this component is over the complexity limit, so this can probably be removed?